### DISCUSSION TITLE
winget update releases

### DISCUSSION THREAD

**Discussion by natiT** [gerardog/gsudo#339] [Q&A] [answered]
Hello,

I just want to ask why the winget release is still in version v2.0.4. Is winget not supported anymore? Or do i look into the wrong repository? 
https://winget.run/pkg/gerardog/gsudo

Kind Regards

**Comment by gerardog** [comment, marked as answer]
Hi,
The web site you linked is not an official part of winget. It is outdated, so don't consider it as a primary source.

On the [winget package repository](https://github.com/microsoft/winget-pkgs/tree/master/manifests/g/gerardog/gsudo), gsudo is at v2.4.2, which is only two releases behind of this [github release page](https://github.com/gerardog/gsudo/releases/) currently at v2.4.4. 

The reason for the two releases delay is that: 1) The winget releaser is currently failing, it does not support the portable version introduced in #271. And 2) I released two versions on monday.

Regards, Gerardo.-
